bin/ObjectExtractor

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import os
+import sys
+from argparse import ArgumentParser
+
+from object_extractor import Extractor, FRONTALFACE_DEFAULT
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument('-f', '--file', dest='image_path', help='filename.png')
```

Comment:
Do you think you should add [this](https://docs.python.org/2/library/argparse.html#required)?